TowerForge/TowerForge/Networking/RankingNetwork/GameRankData.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  GameRankData.swift
+//  TowerForge
+//
+//  Created by Vanessa Mae on 08/04/24.
+//
+
+import Foundation
+
+struct GameRankData: Codable {
+    var userId: String
+    var username: String
+    var score: Double
```

Comment:
We might need to think of a different way to store rank information as opposed to storing the score itself directly, but no worries I can handle that on my side. This should work sufficiently for now.